docs/diagrams/business_models.puml

```diff
@@ -0,0 +1,69 @@
+@startuml BusinessModels
+
+object event
+object day
+object room
+object timeslotSize
+object timeslot
+object session
+object style
+object sponsor
+
+event : name = "Name of the Event/Conference"
+event : description = "Description of the Event/Conference"
+event : location = "Location of the Event/Conference"
+event : details = "Markdown introduction of the Event/Conference"
+event : image = "Image of the Event/Conference"
+event : url = "URL of the Event/Conference"
+event : startDate = "Start Date of the Event/Conference"
+event : endDate = "End Date of the Event/Conference"
+event : style = style
+event : days = day[]
+event : timeslotSizes = timeslotSizes[]
+event : sponsors = sponsor[]
+
+sponsor : name = "Name of the sponsor"
+sponsor : image = "Logo of the sponsor"
+sponsor : url = "URL of the sponsor"
+
+day : date = "Date of the Day"
+day : note = "A note for special days"
+day : rooms = room[]
+
+room : name = "Name of the Room"
+room : timeslots = timeslot[]
+
+timeslotSize : size = integer
+timeslotSize : name = "Name of the Timeslot size (e.g. short break, lunch, middle size session, ...)"
+timeslotSize : timeslots = timeslot[]
+
+timeslot : size = timeslotSize
+timeslot : startDate = "Start Date of the Timeslot"
+timeslot : endDate = "End Date of the Timeslot"
+timeslot : room = room
+
+session : title = "Title of the Session"
+session : description = "Description of the Session"
+session : speakers = "Speakers of the Session"
+session : timeslot = timeslot
```

Comment:
`session : isKeynote = "Bool if session is a keynote"`

Reply:
würde eher vorschlagen:
```
session : isBlocking = "Bool if session don't allow parallel sessions"
session : isBreak = "This session is for all rooms like lunch breaks"
session : isNoBlockingBreak = "This session is for all rooms but more as overlay like coffee break"
```